### ISSUE TITLE
bindTextureImage: clearing GL error: 0x501

### ISSUE THREAD

**Issue by dennyabrain** [yulu/ShaderCam#2]
Thank you for sharing this code and documenting your process.
I keep getting the following error on running the code : 
[SurfaceTexture-1-13621-0] bindTextureImage: clearing GL error: 0x501
<core_glGetAttribLocation:633>: GL_INVALID_VALUE
Could not get attrib location for uTransformM
Could not get attrib location for uOrientationM
Could not get attrib location for ratios
Could not get attrib location for aPosition

basically, both glGetAttribLocation and glGetUniformLocation return -1 all the time.
Would you know what I could do to fix this?

Thank you

**Comment by dennyabrain** [comment]
Oh my bad.. 
Turns out i had a auto generated line - #version 120 in my vertex shader and it was unsupported.

Thanks!
